### ISSUE TITLE
[Bug] Why isn't my Oresim working? 

### ISSUE THREAD

**Issue by ArthurLo3371** [AntiCope/meteor-rejects#265]
### Describe the bug

My seed and version is both for 1.20 but the Oresim module says that it isn't. Help pls

### Crashlog

![image](https://github.com/AntiCope/meteor-rejects/assets/79695498/04527feb-2e3a-474e-afd8-172674ed0d59)

### Steps to reproduce

1. Type in seed using `.seed <seed> 1.20
2. Activate Oresim

### Operating System

Windows

### Addon Version

-0.3

### Meteor Version

0.5.4-1857

### Minecraft Version

1.20.1

### Other mods

```plaintext
1.SeedcrackerX
```

**Comment by CBTPlxiity** [comment]
yeah the same is happening with mine did you maybe figure it out?

**Comment by jklmm7xw** [comment]
> yeah the same is happening with mine did you maybe figure it out?

type .seed (seed) 1.19 and it will work
